airflow/operators/s3_file_transform_operator.py

```diff
@@ -100,10 +99,9 @@ def execute(self, context):
                 )
             self.log.info("Uploading transformed file to S3")
             f_dest.flush()
-            dest_s3.load_file(
+            dest_s3.upload_file(
```

Comment:
`dest_s3` is an S3Hook object -- the method is called `load_file`, `upload_file` does not exist.

Reply:
You're right :) Fixing.